samples/bluetooth/broadcast_audio_assistant/src/main.c

```diff
@@ -369,11 +468,33 @@ static void bap_broadcast_assistant_add_src_cb(struct bt_conn *conn, int err)
 	}
 }
 
+static void pa_sync_synced_cb(struct bt_le_per_adv_sync *sync,
+			      struct bt_le_per_adv_sync_synced_info *info)
+{
+	if (sync == pa_sync ||
+	    (info != NULL && bt_addr_le_eq(info->addr, &selected_addr) &&
+	     info->sid == selected_sid)) {
```

Comment:
```suggestion
	if (sync == pa_sync) {
```

Just checking the sync object should be fine